server/datastore/psql/all_maintenance_requests.go

```diff
@@ -0,0 +1,18 @@
+package psql
+
+import(
+	// Import the necessary packages
+)
+
+/* AllMaintenanceRequests performs a SQL query to fetch all rows in the
+*  maintenance_requests table */
+func AllMaintenanceRequests() {
```

Comment:
you should make this a method (as in, this function is called on a struct) so that you can access the `conn` field directly. The function signature should look like this:

```go
func (db *DB) AllMaintenanceRequests() {
...
}
```

that way you can call `db.conn` directly